### PR TITLE
adds egghead video embeds to docs

### PR DESCRIPTION
**Pull request by zacjones93**
# Changes Made
- adds egghead lesson embeds in place of links

<details>
    <summary>Screenshot of section closed</summary>
    <br>
    
</details>
<details>
    <summary>Screenshot of section open with Embed showing</summary>
    <br>
    
</details>

### CODE REVIEW

File: docs/best/devtools.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # MobX + React DevTools
 
-<a style="color: white; background:green;padding:5px;margin:5px;border-radius:2px" href="https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx">egghead.io lesson 2: devtools</a>
+<details>
+    <summary style="color: white; background:green;padding:5px;margin:5px;border-radius:2px">egghead.io lesson 2: devtools</summary>
```
Comment (zacjones93) [inline docs/best/devtools.md]: Used the same styles applied to `<a>` tags for the dropdowns

File: docs/refguide/observer-component.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 # @observer
 
-<a style="color: white; background:green;padding:5px;margin:5px;border-radius:2px" href="https://egghead.io/courses/manage-complex-state-in-react-apps-with-mobx">egghead.io lesson 1: observable & observer</a>
```
Comment (zacjones93) [inline docs/refguide/observer-component.md]: Not sure if this link was intended to be the course page, I updated to link to the relevant lesson

Comment (mweststrate) [reply docs/refguide/observer-component.md]: Nice catch!

**Comment by mweststrate** [comment]
This is awesome, thanks a lot!

**Pull request merged**